.eslintrc.json

```diff
@@ -502,14 +502,36 @@
                 "ext/js/general/object-property-accessor.js",
                 "ext/js/general/regex-util.js",
                 "ext/js/general/text-source-map.js",
+                "ext/js/language/ar/language-arabic.js",
```

Comment:
I think using wildcards like `language/**/*.js` would be better here but couldn't get the test to pass.

Reply:
Yeah, that fixes one of the tests, but `eslint-config.test.js` in `test-code` I can't figure out